savant-version.iml

```diff
@@ -10,26 +10,26 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="Groovy 2.4" level="application" />
+    <orderEntry type="library" name="groovy-4.0.22" level="application" />
```

Comment:
`Groovy X.X` is sort of the standard we have all across our dev environment. We may want to keep this unless there is a technical benefit to changing it.

Reply:
Agreed. Fixed.